### DISCUSSION TITLE
how to install adminer

### DISCUSSION THREAD

**Discussion by lamsungub** [markshust/docker-magento#425] [General]
I tried to install it by 

1. access adminer.org => coppy "link download"
2. in file src/nginx.conf add adminer after health_check ( ex: location ~ ^/(index|get|static|errors/report|errors/404|errors/503|health_check|adminer)\.php$ { ......}
3. cd src/pub => run wget "link dowload"
4. rename to adminer.php
5. access mylocal.test/adminer.php  

but ,the result is 404 error:page not fount . how to fix it, tkanks

**Comment by rangerz** [comment]
You need put in `pub` folder in container.

into `./bin/bash` to do the same thing.

and `bin/restart`
